qa/L0_long_running_stress/stress.py

```diff
@@ -634,13 +643,7 @@ def stress_thread(name, seed, test_duration, correlation_id_base,
 
                 choice = rng.rand()
 
-                # no-start cannot follow no-end since the server will
-                # just assume that the no-start is a continuation of
-                # the no-end sequence instead of being a sequence
-                # missing start flag.
-                if ((last_choice != "sequence_no_end") and
```

Comment:
I think that this code is not longer working as before because you added non-sequence cases below that set last_choices. You should rename last_choices to last_seq_choices and set it only for the sequence cases and then this original logic should be ok. You will also need a 'last_choice that you set everywhere that you can use when calling count_failed_test_case.

Reply:
I see, will make the changes.